std/http/cookie.ts

```diff
@@ -32,11 +32,14 @@ export interface Cookie {
 
 export type SameSite = "Strict" | "Lax" | "None";
 
+const FIELD_CONTENT_REGEXP = /^[^()@<>,;:\\"\[\]?={}\s]+$/;
```

Comment:
Restrict to ASCII, excluding control characters.
```suggestion
const FIELD_CONTENT_REGEXP = /^(?=[\x20-\x7E]*$)[^()@<>,;:\\"\[\]?={}\s]+$/;
```